.travis.yml

```diff
@@ -73,6 +73,7 @@ jobs:
     - php: 7.0
       env: MOODLE_BRANCH=MOODLE_35_STABLE
     - stage: Deploy
+      if: tag IS present
```

Comment:
Aren't this and the `tags: true` below redundant?  Note I've not looked docs.

Reply:
Aha, so the condition is to skip the whole stage. So we save ourselves from running `before_deploy` (make) completely.

Reply:
Yes, this `if` condition is to skip whole stage (like you see in the build for this PR). The stage will only be used when build has been triggered by tag (either tag was pushed or created via interface). That `on: tags` is condition for deployment itself, probably it is redundant as stage will never be reached in normal PR or master builds, but I would keep it :)

Reply:
Yeah, makes sense, thanks!